descriptor.md

```diff
@@ -43,6 +43,8 @@ The following fields contain the primary properties that constitute a Descriptor
 
     This OPTIONAL property contains arbitrary metadata for this descriptor.
     This OPTIONAL property MUST use the [annotation rules](annotations.md#rules).
+ 
+Descriptors pointing to [`application/vnd.oci.image.manifest.v1+json`](manifest.md) SHOULD include the extended field `platform`, see [Image Index Property Descriptions](https://github.com/qianzhangxa/image-spec/blob/master/image-index.md#image-index-property-descriptions) for details.
```

Comment:
Thanks @AkihiroSuda. I have updated the code.

And I think we still need "if its target is platform-specific." in `image-index.md` since the manifest in a image index can be of other media type (not just `application/vnd.oci.image.manifest.v1+json`) in which case `platform` can be absent.